articles/azure-functions/functions-create-function-linux-custom-image.md

```diff
@@ -342,13 +342,13 @@ After the binding is defined, you can use the `name` of the binding to access it
 In the root folder, run the [docker build](https://docs.docker.com/engine/reference/commandline/build/) command again, and this time update the version in the tag to `v1.0.2`. As before, replace `<docker-id>` with your Docker Hub account ID. 
 
 ```bash
-docker build --tag <docker-id>/mydockerimage:v1.0.0 .
```

Comment:
The paragraph above indicates updating the version of the image

Reply:
I wonder if I should fix the image instead? Does it seem weird to start with 1.0.2?